EIPS/eip-3026.md

```diff
@@ -1,7 +1,7 @@
 ---
 eip: 3026
 title: BW6-761 curve operations
-author: Youssef El Housni (@yelhousni), Michael Connor (@iAmMichaelConnor), Aurore Guillevic <aurore.guillevic@inria.fr>
+author: Youssef El Housni (@yelhousni), Michael Connor (@iAmMichaelConnor), Aurore Guillevic <aurore.guillevic@inria.fr>, hujw77 <hujw77@gmail.com>
```

Comment:
```suggestion
author: Youssef El Housni (@yelhousni), Michael Connor (@iAmMichaelConnor), Aurore Guillevic <aurore.guillevic@inria.fr>, hujw77 (@hujw77)
```
you probably want this as it will allow you to merge changes to the EIP w/o waiting for approval from the other authors.

Reply:
Yep, I want to become one of the author in EIP-3026.